possel/data/frontend/README.md

```diff
@@ -0,0 +1,18 @@
+This readme is to help those working on the frontend.
+
+There are several dependencies that need to be included.
+
+They can be installed as followed:
+
+    gem install jekyll
+    npm install
```

Comment:
> ubuntu
There's your problem.

But seriously, jekyll is annoying to install but is only needed to
compile/serve the static UI. Hopefully all dependence on Jekyll will be
removed soon

On Fri, 27 Nov 2015 at 16:22 Kit Barnes <notifications@github.com> wrote:

> In possel/data/frontend/README.md
> <https://github.com/possel/possel/pull/22#discussion_r46059430>:
>
> > @@ -0,0 +1,18 @@
> > +This readme is to help those working on the frontend.
> > +
> > +There are several dependencies that need to be included.
> > +
> > +They can be installed as followed:
> > +
> > +    gem install jekyll
> > +    npm install
>
> I haven't been able to make this work nicely on ubuntu
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/possel/possel/pull/22/files#r46059430>.
>